server/src/main/kotlin/org/javacs/kt/overridemembers/OverrideMembers.kt

```diff
@@ -0,0 +1,282 @@
+
+        val codeAction = CodeAction()
+        codeAction.edit = WorkspaceEdit(mapOf(uri to listOf(textEdit)))
+        codeAction.title = member
+
+        codeAction
+    }
+}
+
+// TODO: any way can repeat less code between this and the getAbstractMembersStubs in the ImplementAbstractMembersQuickfix?
+private fun getUnimplementedMembersStubs(file: CompiledFile, kotlinClass: KtClass): List<String> =
+    // For each of the super types used by this class
+    // TODO: does not seem to handle the implicit Any and Object super types that well. Need to find out if that is easily solvable. Finds the methods from them if any super class or interface is present
+    kotlinClass
+        .superTypeListEntries
+        .mapNotNull {
+            // Find the definition of this super type
+            val referenceAtPoint = file.referenceExpressionAtPoint(it.startOffset)
+            val descriptor = referenceAtPoint?.second
+            val classDescriptor = getClassDescriptor(descriptor)
+
+            // If the super class is abstract, interface or just plain open
+            if (null != classDescriptor && classDescriptor.canBeExtended()) {
+                val superClassTypeArguments = getSuperClassTypeProjections(file, it)
+                classDescriptor
+                    .getMemberScope(superClassTypeArguments)
+                    .getContributedDescriptors()
+                    .filter { classMember ->
+                        (classMember is FunctionDescriptor &&
+                         classMember.canBeOverriden() &&
+                         !overridesDeclaration(kotlinClass, classMember)) ||
+                        (classMember is PropertyDescriptor &&
+                         classMember.canBeOverriden() &&
+                         !overridesDeclaration(kotlinClass, classMember))
+                    }
+                    .mapNotNull { member ->
+                        when (member) {
+                            is FunctionDescriptor -> createFunctionStub(member)
+                            is PropertyDescriptor -> createVariableStub(member)
+                            else -> null
+                        }
+                    }
+            } else {
+                null
+            }
+        }
+        .flatten()
+
+private fun ClassDescriptor.canBeExtended() = this.kind.isInterface ||
+    this.modality == Modality.ABSTRACT ||
+    this.modality == Modality.OPEN
+
+private fun FunctionDescriptor.canBeOverriden() = (Modality.ABSTRACT == this.modality || Modality.OPEN == this.modality) && Modality.FINAL != this.modality && this.visibility != DescriptorVisibilities.PRIVATE && this.visibility != DescriptorVisibilities.PROTECTED
+
+private fun PropertyDescriptor.canBeOverriden() = (Modality.ABSTRACT == this.modality || Modality.OPEN == this.modality) && Modality.FINAL != this.modality && this.visibility != DescriptorVisibilities.PRIVATE && this.visibility != DescriptorVisibilities.PROTECTED
```

Comment:
I _think_ we can do

```suggestion
private fun MemberDescriptor.canBeOverriden() = (Modality.ABSTRACT == this.modality || Modality.OPEN == this.modality) && Modality.FINAL != this.modality && this.visibility != DescriptorVisibilities.PRIVATE && this.visibility != DescriptorVisibilities.PROTECTED
```

Haven't checked whether that compiles though. That would let us get rid of some code duplication here and there.